docs/reference/analysis/charfilters/pattern-replace-charfilter.asciidoc

```diff
@@ -257,14 +257,14 @@ GET my_index/_search
 }
 ----------------------------
 // CONSOLE
-// TEST[skip:The value for took is always different]
 
 The output from the above is:
 
 [source,js]
 ----------------------------
 {
   "timed_out": false,
+  "took": $body.took,
```

Comment:
I usually put a number here and use the replacement below to put the "$body.took" bit in. I usually use however long it took on my laptop so a particularly savvy ready gets a sense of how long it might take.